ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -216,19 +217,23 @@ interface State {
   workspaceDetails?: WorkspaceAdminView;
   cloudStorageTraffic?: CloudStorageTraffic;
   loadingData?: boolean;
+  loadingLockUnlockStatus: boolean;
   runtimeToDelete?: ListRuntimeResponse;
   confirmDeleteRuntime?: boolean;
   dataLoadError?: Response;
   files?: Array<FileDetail>;
+  showLockWorkspaceModal: boolean;
 }
 
-class AdminWorkspaceImpl extends React.Component<Props, State> {
+export class AdminWorkspaceImpl extends React.Component<Props, State> {
```

Comment:
why this change?

Reply:
I was writing test for this class but realized its going to take some time. I added export  just for that